packages/system/src/components/homepage/contributor-banner.tsx

```diff
@@ -33,8 +33,16 @@ export function ContributorBanner({
 					Create a PR if you see mistakes, room for improvement, or new
 					opportunities to help the dev team.
 				</p>
+				<Button
+				color="primary"
+				size="large"
+				as="a"
+				href="https://github.com/thisdot/framework.dev"
+			    >
+				Contribute
+			    </Button>
```

Comment:
Formatting is still off here.
Remember that all of the elements on the page have to follow the same consistent formatting. 

Also, I would suggest installing [Prettier](https://prettier.io/) in your code editor which will handle all formatting for your projects. 👍